lib/interface/cli/helpers/general.js

```diff
@@ -106,15 +106,26 @@ const prepareKeyValueObjectsFromCLIEnvOption = (environmentVariables) => {
     return variables;
 };
 
+/**
+ * will return an array of objects { key, value, encrypted } from parsing an array of objects: key=value ,key: {value: "value", encrypted: true }
+ * @param environmentVariables
+ * @returns Array of { key, value, encrypted }
+ */
 const prepareKeyValueObjectsFromEnvFileOption = (environmentVariables) => _.map(environmentVariables, (value, key) => {
     if (_.isObject(value)) {
-        const { val, encrypted } = value;
-        return { key, value: val, encrypted };
+        return { key, ...value };
```

Comment:
since it's only 2 properties, I'd rather to have them defined explicitly than the spread operator, similar to what you had before (to improve readability and to avoid passing any unexpected properties since its JS)